templates/dart/docs/example.md.twig

```diff
@@ -22,7 +22,7 @@ void main() { // Init SDK
 
 {% endif %}
 {% if parameter.required %}
-    {{ parameter.name | caseCamel | overrideIdentifier }}: {{ parameter | paramExample | replace({'<field-name>': (parameter.name | caseCamel) }) | raw }},
+    {{ parameter.name | caseCamel | overrideIdentifier }}:{% if parameter.enumValues | length > 0%} {{parameter.enumName}}.{{(parameter.enumKeys[0] ?? parameter.enumValues[0]) | caseCamel | replace({'-': ''}) }}.value{% else%}{{ parameter | paramExample | replace({'<field-name>': (parameter.name | caseCamel) }) | raw }} {% endif %},
```

Comment:
Let's make a global twig filter `stripEnum` that will handle the replacements for us so we don't need to remember them for every template